lib/src/androidTest/java/at/bitfire/vcard4android/contactrow/EventBuilderTest.kt

```diff
@@ -140,6 +142,17 @@ class EventBuilderTest {
         }
     }
 
+    @Test
+    fun testType_BirthdayWithTimezone() {
```

Comment:
`testType_Birthday` means that the test checks for whether "type: birthday" in the `Contact` is correctly interpreted as `Event.TYPE_BIRTHDAY` in the generated contacts provider data row.

However we don't test for the translation of "type" here, but we test whether date/times with time zones are correctly translated into the corresponding `Event.START_DATE` in the contacts provider data row.

So this test should be named `testStartDate_WithOffset` (_offset_ because it's not really a time zone, but only a +hhmm offset). (And it should be groups to the other `testStartDate` tests and within the group alphabetically.)